### ISSUE TITLE
NPE in AvatarUtils

### ISSUE THREAD

**Issue by cheah** [intercom/intercom-android#163]
```
Fatal Exception: java.lang.NullPointerException
       at io.intercom.android.sdk.utilities.AvatarUtils.getRoundedBitmap(AvatarUtils.java:67)
       at io.intercom.android.sdk.gcm.GcmNotificationBuilder.getBitmapFromURL(GcmNotificationBuilder.java:106)
       at io.intercom.android.sdk.gcm.GcmNotificationBuilder.generateAvatar(GcmNotificationBuilder.java:116)
       at io.intercom.android.sdk.gcm.GcmNotificationBuilder.getAppIcon(GcmNotificationBuilder.java:127)
       at io.intercom.android.sdk.gcm.GcmNotificationBuilder.createSingleNotification(GcmNotificationBuilder.java:42)
       at io.intercom.android.sdk.gcm.GcmNotificationManager.createNotification(GcmNotificationManager.java:42)
       at io.intercom.android.sdk.gcm.GcmIntentService.onHandleIntent(GcmIntentService.java:54)
       at android.app.IntentService$ServiceHandler.handleMessage(IntentService.java:65)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:146)
       at android.os.HandlerThread.run(HandlerThread.java:61)
```

**Comment by harkin** [comment]
@cheah could you give me some information on what phones you're seeing this crash on and it's frequency?

**Comment by cheah** [comment]
http://crashes.to/s/6dfaeea6d6a

**Comment by DaleCantwell** [comment]
This should be solved in the new version 1.1.13
